tests/test_scripts.py

```diff
@@ -82,3 +84,123 @@ def test_proto_conversion(tmpdir):
     assert p.returncode == 0
 
     assert p.stdout.startswith(b'{\n  "meta": ') or p.stdout.startswith(b'{\r\n  "meta": ')
+
+
+def run_detect_duplicate_features(rule_dir, rule_path):
```

Comment:
this can added here: https://github.com/mandiant/capa/pull/1463/files#diff-77d9a7b27446244085674121ddc1e16fa75a37e2b4c8c27cf5ab8fcffedea7aaL44

Reply:
You mean to create a new set of parameters [script , args, expected_overlaps] and add it to pytest and use `run_program` on it

Reply:
correct

Reply:
I don't think it is suitable to add a new set of parameters, since for the test new rule files need to be created at first and only then can pytest run the `test_scripts` function with rule_paths as args in test_scripts.py